torchvision/datasets/voc.py

```diff
@@ -218,6 +218,9 @@ def parse_voc_xml(self, node):
             for dc in map(self.parse_voc_xml, children):
                 for ind, v in dc.items():
                     def_dic[ind].append(v)
+            if node.tag == 'annotation':
+                if isinstance(def_dic['object'], object):
```

Comment:
Wait, isn't `isinstance(..., object)` always true?

Reply:
Nop, when the annotation only contains one object to be localized the XML parser parses this property as an object. When it has multiple objects. it is parsed as a list.

Reply:
@joaqo wait a minute,
```python
isinstance([], object) == True
```
right?

Reply:
You are correct @fmassa . Strangely in the tests I made it was working and the values that were already lists were not being made lists of lists `[[]]` 
Nevertheless I'll the code for the converse - check if it is not a list.